src/arrayMethodSort.js

```diff
@@ -3,9 +3,21 @@
 /**
  * Implement method Sort
  */
+
 function applyCustomSort() {
-  [].__proto__.sort2 = function(compareFunction) {
-    // write code here
+  const defaultCompareSortFunction = (a, b) => String(a) > String(b) ? 1 : -1;
+  [].__proto__.sort2 = function (compareFunction = defaultCompareSortFunction) {
```

Comment:
```suggestion
  const defaultCompareSortFunction = (a, b) => String(a) > String(b) ? 1 : -1;
  
  [].__proto__.sort2 = function (compareFunction = defaultCompareSortFunction) {
```

Add a blank line to improve code readibility